routes/admin.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+const { isLoggedIn, isAdmin } = require('./utils/middleware') ;
+const user = require('./model/usermodel');
+
+
+
+// router.delete('/admin/users',(req,res,next)=>{
+//     user.remove({},(err,user)=>{
+//         res.json(user);
+//     })
+// });
+
+router.get("/admin/addCompany" , (req,res) => {
+	res.send("foobar");
+});
+
+
+router.get("/admin/deleteCompany" , (req,res) => {
+	res.send("foobar");
+});
+
+
```

Comment:
These routes should be post and delete routes

Reply:
Also use IsAdmin and isLoggedIn middleware on these routes so that only admins can make requests on these url.